source/gent/tier2_hardware.rst

```diff
@@ -18,17 +18,20 @@ Compute clusters
  =============== ========== =========================================================== ===================== =========================== ======================= ===================
   swalot              128    2 x 10-core Intel E5-2660v3 (Haswell-EP @ 2.6 GHz)         116 GiB               1 TB                        FDR InfiniBand          CentOS 7
```

Comment:
Can you update the OS of swalot as well?

Reply:
done